File: KassenSystemAngular/ClientApp/src/app/shared/checkout-item.model.ts
export class CheckoutItem {
  id: number;
  itemId: number;
  name: string;
  priceSingle: number;
  priceFull: number;
  amount: number;
  constructor() {
    //this.id = 0;
    this.itemId = 0;
    this.name = "";
    this.priceSingle = 0;
    this.priceFull = 0;
    this.amount = 0;
  }
}

File: KassenSystem/Models/CheckoutItemModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace KassenSystem.Models
{
    public class CheckoutItem
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public int ItemId { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public Decimal PriceSingle { get; set; }
        [Required]
        public Decimal PriceFull { get; set; }
        [Required]
        public int Amount { get; set; }

    }
}
